modules/rosa-sdpolicy-security.adoc

```diff
@@ -0,0 +1,48 @@
+
+- GitHub or GitHub Enterprise
+- GitLab
+- Google
+- LDAP
+- OpenID connect
+
+== Privileged containers
+Privileged containers are not supported on {product-title}. To enable Red Hat to operate {product-title} as a managed service with an SLA, some restrictions are enforced to limit the ability of rogue or accidental changes that could impact the service.
+
+== Customer administrator user
+In addition to normal users, {product-title} provides access to an {product-title}-specific Group called `dedicated-admin`. Any users on the cluster that are members of the `dedicated-admin` group:
+
+- Have administrator access to all customer-created projects on the cluster
+- Can manage resource quotas and limits on the cluster
+- Can add and manage `NetworkPolicy` objects
+- Are able to view information about specific nodes and PVs in the cluster, including scheduler information
+- Can access the reserved `dedicated-admin` project on the cluster, which allows for the creation of Service Accounts with elevated privileges and also gives the ability to update default limits and quotas for projects on the cluster.
+
+== Cluster administration role
+The administrator of {product-title} has default access to the `cluster-admin` role for your organization's cluster. While logged into an account with the `cluster-admin` role, users have increased permissions to run privileged security contexts.
+
+== Project self-service
+All users, by default, have the ability to create, update, and delete their projects. This can be restricted if a member of the `dedicated-admin` group removes the self-provisioner role from authenticated users:
+----
+oc adm policy remove-cluster-role-from-group self-provisioner system:authenticated:oauth
+----
+
+Restrictions can be reverted by applying:
+----
+oc adm policy add-cluster-role-to-group self-provisioner system:authenticated:oauth
+----
+
+== Regulatory compliance
+Refer to link:https://www.openshift.com/products/dedicated/process-and-security#compliance[OpenShift Dedicated Process and Security Overview] for the latest compliance information.
+
+== Network security
+With {product-title} on AWS, AWS does provide a standard DDoS protection on all load balancers, called AWS Shield. This provides 95% protection against most commonly used level 3 and 4 attacks on all the public facing load balancers used for {product-title}. We also add a 10 second timeout for HTTP requests coming to our haproxy router to receive a response or the connection is closed to provide additional protection.
```

Comment:
_... AWS provides a standard DDoS protection on all load balancers called AWS Shield._

Reply:
s/10 second timeout/10-second timeout
Along the lines of Lindseys' comment, also avoid using "our"

Reply:
Done.